Resources/Private/JavaScript/Host/Containers/FooterBar/style.css

```diff
@@ -0,0 +1,4 @@
+.footerBar {}
+.footerBar--hidden {
```

Comment:
Hey @dimaip - Thanks for the work! :)
For consistency reasons, this should be named `.footerBar--isHidden`